apps/examples/package.json

```diff
@@ -33,6 +33,7 @@
 		"e2e-ui": "playwright test --ui -c ./e2e/playwright.config.ts"
 	},
 	"dependencies": {
+		"@babel/core": "^7.23.7",
 		"@babel/plugin-proposal-decorators": "^7.21.0",
```

Comment:
I'm 99% sure we can get rid of these. I don't believe we use babel anywhere now.
```suggestion
```

Reply:
yeah, I wasn't sure it's not used, thanks for confirming it!